docs/devGuide/development/settingUp.md

```diff
@@ -80,4 +80,4 @@ If you ever need to uninstall the hooks, simply run `uninstall` instead of `inst
    Try running either `markbind serve -d` or `npm run build:web` to view frontend changes (especially after pulling a frontend update that someone else may have pushed). You can see more details [here](workflow.md#editing-frontend-features).
 
 {% from "njk/common.njk" import previous_next %}
-{{ previous_next('../bootcamp/implementAFeature', 'workflow') }}
\ No newline at end of file
+{{ previous_next('../bootcamp/implementAFeature', 'workflow') }}
```

Comment:
What's the change here?

Reply:
intelij automatically adds a newline and I thought that adding that was good.